includes/class-wc-payments.php

```diff
@@ -222,6 +222,12 @@ public static function init() {
 		// Priority 5 so we can manipulate the registered gateways before they are shown.
 		add_action( 'woocommerce_admin_field_payment_gateways', [ __CLASS__, 'hide_gateways_on_settings_page' ], 5 );
 
+		// Combine JSON translation files (from chunks) when language packs are updated.
+		add_action( 'upgrader_process_complete', [ __CLASS__, 'combine_translation_chunk_files' ], 10, 2 );
+
+		// Grab translation strings from Webpack-generated chunks.
+		add_filter( 'load_script_translation_file', [ __CLASS__, 'load_script_translation_file' ], 10, 3 );
+
```

Comment:
I wonder if it'd be better to create a class for this custom translation handling instead of adding more things to `class-wc-payments.php`. I don't even think we'd need to add a class, maybe it's ok to just `require_once` a file that sets this up?

I think having some sort of `class-wc-payments-translation-loader.php` (or something) would help with the maintenance of this going forward 🙂 .

Reply:
I agree it would help long term maintenance, I'll move the code to its own class 😄